CHANGELOG.md

```diff
@@ -22,15 +22,27 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 ## New features
 
 * Traits for "metaprogramming" with `seqan3::type_list` and type packs have been added.
+* Support for PowerPC has been added.
 
 ## API changes
 
 ### The `type_list` header has moved
 
-If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
+If you included `<seqan3/core/type_list.hpp>`, you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
+
+### The `seqan3::fm_index` and `seqan3::bi_fm_index` now use strong typed template parameters
+
+Instead of `false` and `true`, [seqan3::text_layout::single](@ref seqan3::text_layout) and
+[seqan3::text_layout::collection](@ref seqan3::text_layout) are now used to indicate single texts and text
+collections, respectively.<br>
+If you used `fm_index<false>` to create an index over a single text, use `fm_index<text_layout::single>`.<br>
+If you used `fm_index<true>` to create an index over a collection, use `fm_index<text_layout::collection>`.<br>
+The changes in `seqan3::bi_fm_index` are analoguous.
 
 ## Notable Bug-fixes
 
+* Copying and moving the `seqan3::fm_index` and `seqan3::bi_fm_index` now work properly.
```

Comment:
We should have some rules for which things are notable :thinking: 

I leaning towards only listing things that were reported by users, but I am not sure about it. Let's discuss at the next strategy meeting!

Reply:
It was reported...so it'd fit. 
We could argue that moving or even copying an index is not the most common use case and the fix is therefore not notable. 
So let's discuss on Monday and postpone this PR until then.